specs/IR-0001/suite/test_spec_ir0001.py

```
@@ -0,0 +1,381 @@
+        "单账号深度报告", "多账号聚合", "SOP",
+        "场景描述", "口播台词", "运镜", "画面建议",
+        "去重机制", "频控", "代理 IP 池",
+        "异步处理队列", "LLM 网关", "热切换",
+        "faster-whisper", "RapidOCR", "LiteLLM", "MediaCrawler", "参考不依赖",
+    ]
+
+    def test_anchor_coverage(self):
+        s = read(SPEC)
+        missing = [a for a in self.ANCHORS if a not in s]
+        self.assertEqual(missing, [], f"语义锚缺失: {missing}")
+        # 锚密度：正文（去 frontmatter）需有体量，防"关键词堆砌+空壳条款"
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)